akka-actor/src/main/java/akka/annotation/ApiMayChange.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Marks APIs that are meant to evolve towards becoming stable APIs, but are not stable APIs yet.
+ * 
+ * Evolving interfaces can change from one feature release to another release (i.e. 2.1 to 2.2) without up-front notice.
```

Comment:
From binary compatibility point yes, but I understand that we want to extend this with something in-between - I understand `ApiMayChange` as _this can change between minor releases contrary to general policy_ . My comment was to highlight that from my point of view the world for users of `akka` and (especially) `akka-http` is a fast moving one, and I (as a consumer of your product :) ) would like to be able to get updates to experimental apis faster than on minor release cycle